controllers/chefs/createChef/index.js

```diff
@@ -0,0 +1,16 @@
+import Chef from '../../../models/chef/Chef.model.js';
+
+export const createChef = async (req, res) => {
+  const userId = '65520e1b49c89850ff1256ea';
+
+  // const newChef = new Chef({ ...req.body, userId: req.user.id });
+  const newChefData = {
+    ...req.body,
+    userId: userId,
+  };
+
+  const newChef = new Chef(newChefData);
+  await newChef.save();
```

Comment:
Also, if I'm not mistaken, you still need to add the role of the chef to the user document in the roles array